tests/modules/services/window-managers/sway/sway-invalid-config.nix

```
@@ -0,0 +1,12 @@
+{ ... }:
+
+{
```

Since this will pull down the sway package closure we'll want to mark it as a big test:

```suggestion
{ lib, ... }:

lib.mkIf config.test.enableBig {
```